### ISSUE TITLE
Moderation of Open edX official forums

### ISSUE THREAD

**Issue by antoviaque** [openedx/community-wg#50]
Moderate https://discuss.openedx.org/

List of moderators: https://discuss.openedx.org/g/moderators

**Comment by antoviaque** [comment]
Who else is a moderator currently?

**Comment by regisb** [comment]
The full list of moderators is available here: https://discuss.openedx.org/g/moderators

**Comment by antoviaque** [comment]
@regisb Thanks! 

I'm adding it to the description then, to not maintain a separate list, but people can feel free to add/remove themselves as assignees here to reflect their responsibilities in tickets & sprint boards.

**Comment by sarina** [comment]
Hey! Before I move forward with organizing moderators, I thought it would be good to define what we mean when we say "forums moderation". I put forth a definition and I'd love your input: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3278995469/WIP+Forums+Moderation+on+Discourse

**Comment by nizarmah** [comment]
@sarina I didn't leave a comment on confluence because I think the role definition looks good 👍🏼 
1 login per week should be enough for moderation; ideally it would be a bit more frequent.

Thanks for creating a slack channel for discourse moderators though. I don't think it existed before, or I might have not been aware of it.

**Comment by sarina** [comment]
@nizarmah thanks! I think 1x/week is on the low end, yes, but we have 15 moderator seats and everyone does at least 1x/week we should have good coverage. I'll add a note that more is encouraged.

**Comment by antoviaque** [comment]
@sarina The rules look great! Thanks for putting them together. I've been doing most of these (and I go review/moderate once per week, so for me this frequency is perfect :p), but I've learned a couple of things reading the doc, so that was a useful read. I'm commenting directly on the page there, inline -- mostly small nits.

**Comment by sarina** [comment]
I am going to close this as done.

https://openedx.atlassian.net/wiki/spaces/COMM/pages/3278995469/Forums+Moderation+on+Discourse has been reviewed and is close to if not at a finalized state.

I messaged all existing mods and removed those unable to continue moderating. I made sure to add any additional people to the Core Contributor page (https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Current+Open+edX+Core+Contributors). I messaged everyone the commitment, made sure the Staff category worked on Discourse, and made sure the private `#discourse-moderation` Slack channel has the correct membership.

From here, I think we are ready to start recruiting more moderators. A goal of 5 new moderators would be ideal, in my mind. I think that should be a separate ticket.
